Fabric.Identity.API/Validation/DelegationGrantValidator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using IdentityModel;
+using IdentityServer4.Models;
+using IdentityServer4.Validation;
+
+namespace Fabric.Identity.API.Validation
```

Comment:
Should we consider moving this to a new namespace called Fabric.Identity.API.ExtensionGrantValidators? While it is named *Validator, its not really synonymous with our other validators that validate API models.

Reply:
Yes, I agree. Will move this as well.